### ISSUE TITLE
Confirmation on the descriptor_evaluation flow

### ISSUE THREAD

**Issue by ckcraig01** [rpautrat/SuperPoint#59]
Dear Remi:

  Thank you so much for building up this repo!

1. I downloaded the Hpatches full sequences from this site:
https://github.com/hpatches/hpatches-dataset
HPatches full sequences [1.3GB].

2. My command is:
python export_descriptors.py configs/classical-descriptors.yaml orb_v --export_name=orb_hpatches-v

Then I use "descriptors_evaluation_on_hpatches_test.ipynb" to check the result.

(Note: I can produce the same result with you with "detector" evaluation + harris)

getting results of orb_hpatches-v: 0.10508474576271186 instead of 0.129 as you reported.

Would like to confirm if I have any incorrect settings? Thank you!

And this is my detailed config:

data:
    name: 'patches_dataset'
    dataset: 'hpatches'  # 'hpatches' 'coco'
    alteration: 'v'  # 'i' 'v' 'all'
    cache_in_memory: false
    validation_size: 100
    preprocessing:
        resize: [480, 640]  # False for coco
model:
    name: 'classical_detectors_descriptors'
    method: 'orb'  # 'orb' 'sift'
    batch_size: 50  # unused
    learning_rate: 0.001  # unused
    nms: 8
    top_k: 1000
eval_iter: 600
seed: 1

**Comment by rpautrat** [comment]
It seems that you have exactly the same parameters as me, so this result is a bit weird indeed. Is there anything stochastic in ORB features? It might come from this.
At least the difference is not significant for the comparison with SIFT and SuperPoint, so it is indeed probably due to a stochastic parameter somewhere.
